app-arch/unshield/unshield-1.6.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2025 Gentoo Authors
```

Comment:
```
if(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/test")
    message(STATUS "BUILD_TESTING: ${BUILD_TESTING}")
else()
    set(BUILD_TESTING OFF)
endif()
```

Let's wire that up (i.e. pass `-DBUILD_TESTING=$(usex test)` if it makes a difference to targets built with and without tests, or makes tests actually do something in src_test)?

Reply:
No tests, included in the release tar ball. Explicitly disabled the tests ind configure phase and made a comment to check in future releases if upstream changed their mind.